package.json

```diff
@@ -12,7 +12,9 @@
     "prepublishOnly": "git push --follow-tags",
     "prerelease": "npm t",
     "release": "standard-version -s",
-    "test": "tap"
+    "coverage": "tap --coverage",
```

Comment:
As of tap v13, there's no need to set `--coverage` unless it had been disabled by a previous config.  It's on by default.